src/sql/workbench/contrib/dashboard/browser/core/dashboardPage.component.ts

```diff
@@ -254,8 +254,9 @@ export abstract class DashboardPage extends AngularDisposable implements IConfig
 
 		let toolbarActions = [];
 		_tasks.forEach(a => {
-			let iconClassName = TaskRegistry.getOrCreateTaskIconClassName(a);
-			toolbarActions.push(new ToolbarAction(a.id, a.title.toString(), iconClassName, this.runAction, this, this.logService));
+			const iconClassName = TaskRegistry.getOrCreateTaskIconClassName(a);
+			const title = typeof a.title === 'string' ? a.title : a.title.value;
```

Comment:
the type of title could be string or ILocalizedString
![image](https://user-images.githubusercontent.com/13777222/118739165-91767300-b7fd-11eb-838b-8c38a79148fc.png)

in vscode, it is been handled properly like the change I am making here.